compiler-rt/test/ubsan/TestCases/Misc/Posix/diag-stacktrace.cpp

```diff
@@ -2,10 +2,10 @@
 // UNSUPPORTED: target=thumb{{.*}}
 // UNSUPPORTED: android
 
-// RUN: %clangxx -fsanitize=return %gmlt -O2 -fno-omit-frame-pointer -fasynchronous-unwind-tables %s -o %t
+// RUN: %clangxx -Wno-error=return-type -fsanitize=return %gmlt -O2 -fno-omit-frame-pointer -fasynchronous-unwind-tables %s -o %t
```

Comment:
Probably better to land all fix-ups in a separate PR before actual feature, so we don't revert/related unnecessary stuff.

Reply:
Yup, at this point it seems like it’ll be better to split this into another separate pr